tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/MatcherTests.cs

```diff
@@ -179,6 +179,54 @@ public void CustomMatcherThrowsOnUnmatched()
                 sessionForRetrieval.Session.Lookup(differentRequest, HeaderlessMatcher, sanitizers: new List<RecordedTestSanitizer>(), remove: false);
             });
         }
+
+        [Fact]
+        public async Task CustomMatcherMatchesDifferentUriOrder()
+        {
+            RecordingHandler testRecordingHandler = new RecordingHandler(Directory.GetCurrentDirectory());
```

Comment:
can we add a test that validates the parameter is consumed correctly by the Admin code?